app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import NotVaccinatedError
```

Comment:
use 1 import form app.errors instead of 3

Reply:
do it

Reply:
ok, I was hurry and missed two last issues)

Reply:
done